File: frontend/src/types/teamMember.ts
export interface TeamMember {
  id: number;
  name: string;
  role: string;
  photo_path: string | null;
  photo_alt: string | null;
  sort_order: number | null;
  is_active: boolean;
}

export interface TeamMemberCreate {
  name: string;
  role: string;
  photo_path?: string;
  photo_alt?: string;
  sort_order?: number;
  is_active?: boolean;
}

export interface TeamMemberUpdate {
  name?: string;
  role?: string;
  photo_path?: string;
  photo_alt?: string;
  sort_order?: number;
  is_active?: boolean;
}

File: backend/app/model/team_member.py
from sqlalchemy import Column, Integer, String, Boolean
from app.base_class import Base

class TeamMember(Base):
    __tablename__ = 'team_members'

    id = Column(Integer, primary_key=True, autoincrement=True)
    name = Column(String(100), nullable=False)
    role = Column(String(100), nullable=False)
    photo_path = Column(String(255), nullable=True)
    photo_alt = Column(String(100), nullable=True)
    sort_order = Column(Integer, nullable=True)
    is_active = Column(Boolean, nullable=True, default=True)